index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>main</title>
+</head>
+<body>
+    <div class="header1"><h2>We're open and available for takeaway & delivery. Order Now</h2></div>
+    <div class="barmenu">
+        <div class="aside1"><img src="/imgs/logo.svg" alt="logo"></div>

```

Comment:
#1 First comment.

Hi brothers. Could you change the path from `/imgs/logo.svg` to `./imgs/logo.svg`.
It does work in `Windows` but your path doesn't work in Linux, macOS, Github, and many places. By the way, it's windows...